### DISCUSSION TITLE
Synchronize folder and file dates

### DISCUSSION THREAD

**Discussion by baumaeschi** [WinMerge/winmerge#2125] [Q&A] [answered]
When backing up folder/file trees to a cloud or an external hard disk, especially folder modifications dates are often changed to the date/time when the backup took place. The same happens (sometimes) to file modification dates.

Questions: Are there functions in Winmerge to
 a) detect differences in the dates and 
 b) synchronize dates to their original values (preferably without copying the files and folder)

**Comment by sdottaka** [comment]
If you set the compare method to Modified Date as shown below, you can compare files based on modification date and time instead of file contents.
However, feature b) does not exist in WinMerge.

![image](https://github.com/WinMerge/winmerge/assets/98126/c9d0ba3a-a3c5-4b80-8c30-f0ef3ae646d4)

**Comment by baumaeschi** [comment]
Would you mind making this "File Date/Time synchronization from left to right side" a  future enhancement request? Or is this idea completely outside the scope of winmerge?

**Reply by sdottaka** [reply, marked as answer]
A feature request already exists. This feature is a feature that would be nice to have in WinMerge.
https://github.com/WinMerge/winmerge/issues/1970

**Comment by baumaeschi** [comment]
Ok, thank you for the Link. Hope you find the time one day to implement this. Since many backup programs don't maintain the original date/time of folders and files. Would be very helpful if this could be correctet with winmerge, or, of course, another program (do you know one that could do this?)
